src/praisonai-agents/praisonaiagents/bus/bus.py

```
@@ -209,13 +210,13 @@ def publish_event(self, event: Event) -> Event:
         for sub in subscribers:
             try:
                 if sub.is_async:
-                    # Schedule async callback
+                    # Schedule async callback safely
                     try:
                         loop = asyncio.get_running_loop()
                         loop.create_task(sub.callback(event))
```

Ruff (0.15.10)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)